src/coreclr/src/jit/lclvars.cpp

```diff
@@ -1027,6 +1039,14 @@ void Compiler::lvaInitUserArgs(InitVarDscInfo* varDscInfo)
             lvaSetVarAddrExposed(varDscInfo->varNum);
 #endif // !TARGET_X86
         }
+
+        if (opts.IsOSR() && info.compPatchpointInfo->IsExposed(varDscInfo->varNum))
+        {
+            JITDUMP("-- V%02u is osr exposed\n", varDscInfo->varNum);
```

Comment:
nit: "osr" should be all-caps "OSR" in all output (and comments)

Reply:
Thanks, will fix.